### PR TITLE
Add Java Form recognizer June release notes

### PR DESCRIPTION
**Pull request by samvaity**
Add Java Form recognizer release notes

### CODE REVIEW

File: releases/2020-06/java.md [hunk 1]
```diff
@@ -34,12 +36,18 @@ If you have a bug or feature request for one of the libraries, please post an is
...
-Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here are some of the highlights:
+### Form Recognizer ([Changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md#100-beta3-2020-05-06))
+
+Here are some of the highlights:
+
+#### New Features
+
+- Add support to copy custom model from one Form Recognizer resource to another.
+- Add support for Azure Active Directory credential.
+- [Breaking] Rename refactoring for cross-language consistency.
```
Comment (alzimmermsft) [inline releases/2020-06/java.md:47]: There is a header for breaking changes sections.

File: releases/2020-06/java.md [hunk 2]
```diff
@@ -10,22 +10,24 @@ The Azure SDK team is pleased to announce our June 2020 client library releases.
...
 
 To use the GA and beta libraries, refer to the Maven dependency information below, which may be copied into your projects Maven `pom.xml` file as appropriate. If you are using a different build tool, refer to its documentation on how to specify dependencies.
 
 ```xml
-<!-- Insert dependencies -->
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-ai-formrecognizer</artifactId>
+  <version>1.0.0-beta.3</version>
+</dependency>
```
Comment (alzimmermsft) [inline releases/2020-06/java.md:30]: Should the dependency for Text Analytics be added here as well?

File: releases/2020-06/java.md [hunk 3]
```diff
@@ -34,12 +36,18 @@ If you have a bug or feature request for one of the libraries, please post an is
 
 ## Changelog
 
-Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here are some of the highlights:
+### Form Recognizer ([Changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md#100-beta3-2020-05-06))
```
Comment (alzimmermsft) [inline releases/2020-06/java.md:39]: Changelog for Text Analytics?

Comment (samvaity) [reply releases/2020-06/java.md:39]: I would let @mssfang update it for TA. I just wanted to get this base page in for Java devs to start working on their release notes.

Comment (maririos) [reply releases/2020-06/java.md:39]: changelog link for FR is wrong. consider changing it to the estimate release date

**Pull request merged**